core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -4295,6 +4297,14 @@ payment.amazonGiftCard.info=To pay with Amazon eGift Card, you will need to send
   - Amazon eGift Cards can only be redeemed on the Amazon website they were purchased on (e.g., a gift card purchased on amazon.it can only be redeemed on amazon.it)
 payment.mercadoPago.holderId=UserID linked to financial institution. Like phone number or email or CVU.
 
+payment.sbp.info.account=The Faster Payments System (SBP) is an inter-bank funds transfer service in RUSSIA that allows individuals \
```

Comment:
Why use all cap for Russia?

Reply:
I will merge it as it is, if you agree to use "Russia" instead just add it in a new PR...

Reply:
One big concern in the discussion thread was that the name of this payment method (in English) is almost identical to the existing "Faster Payments" payment method. Added "(SBP)" to help it stand out in the list for people looking for is as SBP.  Still, to emphasize to anyone clicking it intending for the other "Faster Payments", I used all caps so they'd understand quickly this is not what they're looking for.  

I certainly can remove the emphasis of RUSSIA if it seems like overkill. Or, add some other text to clearly and unambiguously differentiate the two payment methods (perhaps on both payment methods?). Really wanted to get some input or response from @pazza83 in the discussion thread on this, but he's mostly avoided my pings to give a comment so I was left on my own.

I am working on adding Russian translations, so certainly could change this however the team feels appropriate.

Reply:
Ah ok. Yes I agree that bother methods using a similar name might be confusing. I would though not do it by all caps for Russia, but rather to change the name of the payment method to maybe: SBP-Faster Payments System or Faster Payments System Russia (SBP)

Reply:
Another thought is to change the display name of the existing "Faster Payments" to "Faster Payments (UK)".  I don't think it is a compatibility issue, the Bisq1 accounts identify the associated payment method with an enumeration value, and once created, the display name is always a "custom name."

As for the Russia one, I was trying to avoid too many acronyms in the name, but "SBP" is definitely necessary in the English version or people might not realize that is what it is.  Maybe to do that and still be consistent: "Faster Payments System-SBP (Russia)"?

I have no problem removing the all caps, that's fine.  The existing Faster Payments has no informational pop-up message at all, and further as both payment methods are locked to their respective currencies someone choosing the wrong one isn't going to get far...

Reply:
Yes the display string should not have any impact on anything. So Faster Payments (UK) is ok as well.
"Faster Payments System-SBP (Russia)" - sounds ok to me.

> ...prefixed...

Yes I think that helps ppl to find quickly their relevant method.

> ...no informational pop-up...

Yes we use popups only if needed. Too many anyway....

Reply:
I don't know but I guess it will take a bit longer for the release. In doubt you can make smaller PRs so that we get in at least parts.

Reply:
Faster Payments System-SBP (Russia) works for me also

Reply:
Wiki has been added https://bisq.wiki/Faster_Payments_System_SBP